week-8/Homework/mandatory/3-dog-photo-gallery/app.js

```diff
@@ -0,0 +1,9 @@
+
+
+document.querySelector('.btn').addEventListener('click',async()=>{
+  console.log('hello world')
+  const res = await fetch('https://dog.ceo/api/breeds/image/random')
```

Comment:
Nice use of `async` and `await`. Can you explain exactly what is happening here?